src/MassTransit/Diagnostics/DiagnosticHeaders.cs

```diff
@@ -1,4 +1,5 @@
 #nullable enable
+// ReSharper disable once CheckNamespace
```

Comment:
Can't you just put all these files in the Logging folder and avoid this nonsense?